docs/dev_guide/ceph_s3_tests_guide.md

```diff
@@ -0,0 +1,143 @@
+# Ceph S3 Tests Guide
+
```

Comment:
Add some intro about what this test is used for, also a link to ceph s3-tests repository will be nice: https://github.com/ceph/s3-tests

Reply:
Great idea, I added it. Thanks.